smriprep/workflows/anatomical.py

```diff
@@ -27,17 +28,180 @@
     PatchedLTAConvert as LTAConvert,
 )
 from niworkflows.interfaces.images import TemplateDimensions, Conform, ValidateImage
+from niworkflows.interfaces.nibabel import Binarize
 from niworkflows.utils.misc import fix_multi_T1w_source_name, add_suffix
-from niworkflows.anat.ants import init_brain_extraction_wf
+from niworkflows.anat.ants import (
+    init_brain_extraction_wf, init_atropos_wf, ATROPOS_MODELS, _pop
+)
 from .norm import init_anat_norm_wf
 from .outputs import init_anat_reports_wf, init_anat_derivatives_wf
 from .surfaces import init_surface_recon_wf
 
+from warnings import warn
+
+
+def init_n4_only_wf(name='n4_only_wf',
+                    omp_nthreads=None,
```

Comment:
```suggestion
                    omp_nthreads,
```
Please make this argument mandatory (will require writing name after)